cvat/apps/iam/views.py

```diff
@@ -86,7 +67,7 @@ def __init__(self, get_response):
     def __call__(self, request):
```

Comment:
Do we need the middleware at all? Should we delete it?

Reply:
This middleware allows us to quickly access specified org_id in requests and also efficiently perform "create" actions like [here](https://github.com/opencv/cvat/blob/89e1b21b5b4d6a684c2d1ff1491ab48ac4a5797a/cvat/apps/engine/views.py#L265).